src/frontend/screens/Security/SecurityContext.tsx

```diff
@@ -0,0 +1,29 @@
+//This is temporary until we determine how password is set
+import * as React from "react";
+
+interface SecurityContextType {
+  passIsSet: boolean;
+  setPassIsSet: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const defaultSecurityContext = [false, () => {}];
+
+//@ts-ignore
+export const SecurityContext = React.createContext<SecurityContextType>(
+  //@ts-ignore
+  defaultSecurityContext
+);
```

Comment:
what's the reason for the ignores here?

Reply:
My `SecurityContextType` Typing is very specific, that the default security context type cannot be captured in it. BUT since my context is initialzed here as well, the default isnt ever going to be used. Also this pattern is most likely going to change once we implement it so I think ts-ignoring is the most graceful way of dealing with it right now